### PR TITLE
Create a workshop materials page

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
This PR aims to create a landing page for our current workshop materials
I've organized the workshops into 3 sections:
- Omero
- Software
- Python

I've added brief descriptions of the workshops and added links to both slides/materials and repos.

### CODE REVIEW

File: workshop_materials.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Workshop Materials
+
+This page is meant to act as a collection of the latest Imaging Applications team workshop materials.
```
Comment (psobolewskiPhD) [inline workshop_materials.md:3]: Maybe should add a sentence about how to find upcoming workshops -- link to RIT calendar?

File: workshop_materials.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Introduction to OMERO
+
+[OMERO](https://www.openmicroscopy.org/omero/) is an image data management and hosting solution. This workshop covers how to store, view, share, annotate, and analyze image data in OMERO.  
+No prior experience with OMERO is necessary.
+
+[Workshop materials](https://thejacksonlaboratory.github.io/omero-training-materials/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/omero-training-materials)
+
+### [OMERO for Python Users](#omero-for-python-users-1)
```
Comment (psobolewskiPhD) [inline workshop_materials.md:15]: I thought this fit better under Python so I linked here, but can swap it around the other way.

File: workshop_materials.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+[Fiji](https://fiji.sc) is a distribution of the [ImageJ](https://imagej.net/software/imagej/) open-source software package for analyzing scientific images. This workshop covers the basic concepts of image analysis (e.g. pixels, bit-depth) and then key Fiji concepts including the GUI layout, scale bars, measurements, and stacks. Finally, selected analysis applications/workflows are demonstrated, including segmentation, colocalization, and tracking. 
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/fiji_workshops/IntroFiji.html)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/fiji_workshops)
+
+### Intermediate Fiji: Macros & Scripting
+`<materials in progress>`
+
+### Introduction to napari
```
Comment (psobolewskiPhD) [inline workshop_materials.md:36]: I put this as a software tool and not python, but maybe should cross list like the Omero for Python?

File: workshop_materials.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+Note: These workshops are intended for people who already have some familiarity with Python.
+
+### (Bio)Image Processing with Python
+
+This workshop will cover reading image files into Python to visualize and manipulate the image data using scikit-image, the most prominent image processing library for Python. Then it covers building a basic bioimage processing pipeline to apply basic filters, threshold, and segment objects in an image.  
+
+[Workshop materials](https://thejacksonlaboratory.github.io/image-processing-ia/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/image-processing-ia)
+
+### OMERO for Python Users
```
Comment (psobolewskiPhD) [inline workshop_materials.md:62]: Not sure what order to put this one in, before/after the zarr/dask?

File: workshop_materials.md (hunk 1 above)
Comment (fercer) [reply workshop_materials.md:3]: I'm not sure if a link to the RIT calendar in this public facing repo would be useful.
Probably a link to the calendar should be wherever `this` page is linked to.

File: workshop_materials.md (hunk 2 above)
Comment (fercer) [reply workshop_materials.md:15]: I like the link here to the actual workshop materials in the Python section, keeps it clean.

File: workshop_materials.md (hunk 3 above)
Comment (fercer) [reply workshop_materials.md:36]: The napari workshop is mostly python, so having it in the Python section and linking from here sounds better in my opinion.

File: workshop_materials.md (hunk 4 above)
Comment (govekk) [reply workshop_materials.md:62]: Where it is makes sense to me

**Comment by govekk** [comment]
At least in the GH markdown viewer, the Heading 1s and Heading 2s aren't really visually different enough to come across as sections - maybe adding some lines between the tools and python sections could help break it up?

File: workshop_materials.md (hunk 1 above)
Comment (psobolewskiPhD) [reply workshop_materials.md:3]: Hmm. I was thinking of putting this markdown in Sharepoint, but yeah then the link here in GitHub would block people. I think that's Ok?

**Comment by psobolewskiPhD** [comment]
I agree @govekk wasn't sure how Sharepoint would render it. I can use H1 for the sections, should be more distinct regardless.

**Pull request merged**